.github/workflows/ci.yml

```diff
@@ -40,7 +40,10 @@ jobs:
         pip install -r local-requirements.txt
         pip install .
     - name: Lint
-      run: black --check .
+      run: |
+        black --check .
+        mypy .
+        flake8  playwright tests
```

Comment:
if you have flake8 and black as pre-commit hooks, you can remove them here and simply have `mypy . && pre-commit run --all-files` as lint stage on GH. mypy is an exception, as it should preferably not be used as pre-commit hook as it [won't be able to](https://github.com/pre-commit/mirrors-mypy) perform full analysis

Reply:
Good catch! Will do it probably in a refactoring PR for the CI bots.